sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensor.java

```diff
@@ -163,8 +164,16 @@ private void saveMeasures(SensorContext context,
           }                             
         }
         
-        newCoverage.save();
-        
+        try
+        {
+          newCoverage.save();
+        } catch(Exception ex) {
+          LOG.error("Cannot save measure '{}' : '{}', ignoring measure", filePath, ex.getMessage());
```

Comment:
@jmecosta setting name and flag name do not match? ignoreInvalidMeasures and FAIL_ON_INVALID_MEASURES have invers logic?